.github/workflows/config-template.yaml.j2

```diff
@@ -26,7 +26,13 @@ singleuser:
   storage:
     homeMountPath: "/home/aiida"
     dynamic:
+
+{%- if RELEASE == 'aiidalab-marketplace' %}
+      storageClass: do-block-storage
+{%- else %}
       storageClass: aws-efs
+{%- endif %}
```

Comment:
Can you [inline](https://jinja.palletsprojects.com/en/3.0.x/templates/#if-expression) this, please?